src/Lykke.Bil2.Contract/Common/FeesValidator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Lykke.Bil2.Contract.Common.Exceptions;
+using Lykke.Numerics.Linq;
+
+namespace Lykke.Bil2.Contract.Common
+{
+    internal static class FeesValidator
+    {
+        public static void ValidateFees(IReadOnlyCollection<Fee> fees, bool isRequest)
```

Comment:
I think this method should return ValidationResult instead of throwing different errors in different contexts.
Something like https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.ivalidatableobject.validate?view=netframework-4.7.2#System_ComponentModel_DataAnnotations_IValidatableObject_Validate_System_ComponentModel_DataAnnotations_ValidationContext_